src/Accordion/accordion.spec.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Accordion from './accordion';
+import AccordionItem from '../AccordionItem/accordion-item';
+
+jest.mock('../AccordionItem/accordion-item', () => 'div');
+
+describe('Accordion', () => {
+    it('renders correctly with min params', () => {
+        const tree = renderer.create(
+            <Accordion>
+                <AccordionItem>Fake child</AccordionItem>
+                <AccordionItem>Fake child</AccordionItem>
+            </Accordion>,
+        ).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    it('renders correctly with accordion false', () => {
+        const tree = renderer.create(
+            <Accordion accordion={false}>
+                <AccordionItem>Fake child</AccordionItem>
+                <AccordionItem>Fake child</AccordionItem>
+            </Accordion>,
+        ).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    it('different className', () => {
+        const tree = renderer.create(
+            <Accordion accordion={false} className="testCSSClass">
+                <AccordionItem>Fake Child</AccordionItem>
+                <AccordionItem>Fake Child</AccordionItem>
+            </Accordion>,
+        ).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    it('handleClick function accordion true', () => {
```

Comment:
If it's the default value of `accordion` IMHO this test case shouldn't be named after the value.